trin-bridge/src/constants.rs

```diff
@@ -7,4 +7,7 @@ pub const BATCH_SIZE: u64 = 128;
 // These are only intended to be used by core team members who have access to the nodes.
 // If you don't have access to the PANDAOPS nodes, but still want to use the bridge feature, let us
 // know on Discord or Github and we'll prioritize support for any provider.
-pub const PANDAOPS_URL: &str = "https://archive.mainnet.ethpandaops.io/";
+/// Execution layer PandaOps endpoint
+pub const BASE_EL_ENDPOINT: &str = "https://geth-lighthouse.mainnet.ethpandaops.io";
+/// Consensus layer PandaOps endpoint
+pub const BASE_CL_ENDPOINT: &str = "https://beacon.mainnet.ethpandaops.io";
```

Comment:
I'm assuming that you've tested this out - and removing the trailing `/` from the url didn't affect the http request? Sometimes these http libraries can be very particular about those kinds of things, but off the top of my head I can't remember if this was required or not with `surf`

Reply:
Yep, tested and working fine without trailing `/`.